src/test/javascript/spec/component/iris/settings/iris-course-settings-update.component.spec.ts

```diff
@@ -28,7 +24,7 @@ describe('IrisCourseSettingsUpdateComponent Component', () => {
     const route = { parent: { params: routeParamsSubject.asObservable() } } as ActivatedRoute;
     let paramsSpy: jest.SpyInstance;
     let getSettingsSpy: jest.SpyInstance;
-    let getModelsSpy: jest.SpyInstance;
+    //let getModelsSpy: jest.SpyInstance;
```

Comment:
You mentioned in the PR description that some features will be reenabled later. Is this one of these cases? It's commented out in a few files.

Reply:
Yes it is. This will be reenable 1:1 in the future, which is why it is simple commented out.